server/services/AlertService.js

```diff
@@ -13,15 +13,12 @@
  *   permissions and limitations under the License.
  */
 
-import { CLUSTER } from './utils/constants';
-import { INDEX } from '../../utils/constants';
-
 export default class AlertService {
   constructor(esDriver) {
     this.esDriver = esDriver;
   }
 
-  getAlerts = async (req, h) => {
+  getAlerts = async (context, req, res) => {
```

Comment:
Why do we need `context` if it is not being used?

Reply:
It's required params by the new route handler signature, details [here](https://github.com/elastic/kibana/blob/master/src/core/MIGRATION_EXAMPLES.md#http-routes)

Reply:
I just tested that there will be `Error: Internal Server Error` without it.